apps/deploy-web/src/hooks/useWalletBalance.ts

```diff
@@ -5,26 +5,93 @@ import { TX_FEE_BUFFER } from "@src/config/tx.config";
 import { useChainParam } from "@src/context/ChainParamProvider";
 import { usePricing } from "@src/context/PricingProvider";
 import { useWallet } from "@src/context/WalletProvider";
+import { useBalances } from "@src/queries/useBalancesQuery";
 import { udenomToDenom } from "@src/utils/mathHelpers";
 import { uaktToAKT } from "@src/utils/priceUtils";
 import { useUsdcDenom } from "./useDenom";
 
-export const useTotalWalletBalance = () => {
+export type TotalWalletBalance = {
+  totalUsd: number;
+  balanceUAKT: number;
+  balanceUUSDC: number;
+  totalUAKT: number;
+  totalUUSDC: number;
+  totalDeploymentEscrowUAKT: number;
+  totalDeploymentEscrowUUSDC: number;
+  totalDeploymentEscrowUSD: number;
+  totalDeploymentGrantsUAKT: number;
+  totalDeploymentGrantsUUSDC: number;
+  totalDeploymentGrantsUSD: number;
+};
+
+export type TotalWalletBalanceReturnType = {
+  isLoadingBalances: boolean;
```

Comment:
it's kinda clear from the context it's loading balance. Would this make more sense?
```suggestion
  isLoading: boolean;
```